prow/cmd/deck/static/script.js

```diff
@@ -492,17 +485,13 @@ function createTimeCell(id, time) {
     var momentTime = moment.unix(time);
     var tid = "time-cell-" + id;
     var main = document.createElement("div");
-    var age = momentTime.fromNow();
-    var timeString = momentTime.format('MMM DD YYYY, HH:MM:SS [GMT]ZZ');
-    main.textContent = timeString;
+    main.textContent = momentTime.format('HH:mm:ss');
     main.id = tid;
-    main.setAttribute("data-time", time);
-    main.classList.add("time-cell");
 
     var tooltip = document.createElement("div");
-    tooltip.textContent = age;
+    tooltip.textContent = momentTime.format('MMM DD YYYY, HH:mm:ss [GMT]ZZ');
```

Comment:
"zZZ" should format as "PST-0800" which should be friendlier.

Reply:
Sure, this came from trimming down the default format (though accidentally capitalizing a bit much :-(

Reply:
z has been deprecated 
https://github.com/moment/moment/issues/162

Reply:
Ah, never mind then. It should be UTC instead of GMT.